Update to use api.v2.emissions-api.org (#47)
emissions-api/emissions-api.github.com: examples/deck.gl.md

In this example we explain how to use [deck.gl](https://deck.gl/#/) –
a WebGL-powered framework for data visualization created by [Uber](https://www.uber.com) –

> to visualize the amount of carbon monoxide in the US from a couple of days.

Reviewer: ```suggestion
to visualize the amount of carbon monoxide in the United States from a couple of days.
```

The result will look somewhat like this:

<div id="container" style="height: 500px;"></div>

We are mostly following the [scripting get-started](https://github.com/uber/deck.gl/blob/7.3-release/examples/get-started/scripting/mapbox/index.html) from deck.gl.

> This example is using version 2 of the API, which can be found under [https://api.v2.emissions-api.org](https://api.v2.emissions-api.org).

Reviewer: ```suggestion
This example is using version two of the API, which can be found at [api.v2.emissions-api.org](https://api.v2.emissions-api.org).
```

## GeoJSON Data

Let's take a look at the data first.

> We request all measured carbon monoxide values from the US from – let's say – the first till the fourth of May 2019 using the [geo.json](https://api.v2.emissions-api.org/ui/#/default/emissionsapi.web.get_data) API endpoint and the URL query parameters *begin*, *end* and *country*:

Reviewer: ```suggestion
We request all measured carbon monoxide values of the United States from first to fourth of May 2019 using the [geo.json](https://api.v2.emissions-api.org/ui/#/default/emissionsapi.web.get_data) API endpoint and the URL query parameters *begin*, *end* and *country*:
```

```
https://api.v2.emissions-api.org/api/v2/carbonmonoxide/geo.json
    ?country=US
    &begin=2019-05-01
    &end=2019-05-04
```

> We are using the *country* url query parameter to receive carbon monoxide values from within the border of the US.

Reviewer: ```suggestion
We are using the *country* url query parameter to receive carbon monoxide values from within the border of the United States.
```

> Further, note that we specify Mai 4st as end date. The end date is not being included in the resulting data.

Reviewer: ```suggestion
Further, note that we specify May 4th as end date. The end date is not being included in the resulting data.
```
